connector/connect/src/main/protobuf/spark/connect/relations.proto

```diff
@@ -41,6 +41,7 @@ message Relation {
     Aggregate aggregate = 9;
     SQL sql = 10;
     LocalRelation local_relation = 11;
+    Intersect intersect = 12;
```

Comment:
This conflicts with some other PRs. I will change this if other PRs are merged first.

Reply:
Wait, don't we have a Union type that does this (e.g. from the set operations?)
@amaliujia ?